src/Form/FieldTextarea.js

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import FieldInput from './FieldInput';
+import IconEdit from './icons/IconEdit';
+
+export default class FieldTextarea extends FieldInput {
+  getInputElement(attributes) {
+    let {props} = this;
+
+    let classes = classNames(props.inputClass, props.sharedClass);
+    attributes = this.bindEvents(attributes);
+
+    if (this.isEditing() || props.writeType === 'input') {
+      return (
```

Comment:
@kennyt Yeah, I'll add the render wrapper here too.

@mlunoe You are right, but I am concerned about doing this in the time frame we have... Would you be okay with me creating a JIRA task for this, which can be completed when the installer is further along? Others, feel free to weigh in here.